EAD_Session7/com/metacube/ead/test/EadSession7ApplicationTests.java

```diff
@@ -0,0 +1,60 @@
+package com.metacube.assignment;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import com.metacube.ead.service.MailSender;
+import com.metacube.ead.service.implementation.MockMailSender;
+import com.metacube.ead.service.implementation.SmtpMailSender;
+
+/**
+ * Test class for EAD Session 7 implementations 
+ * 
+ * @author Aman Gautam
+ * 
+ * Dated : 11/9/19
+ *
+ */
+public class EadSession7ApplicationTests {
+
+	@Autowired
+	private MailSender mailSender ;
+	
+	private void dummyTypeConstructor(MailSender mailSender) {
+		this.mailSender = mailSender;
+	}
+	
+	private void dummyTypeSetter(MailSender mailSender) {
+		this.mailSender = mailSender;
```

Comment:
Why two method for same thing ?

Reply:
dummyTypeSetter() represents the setterMethod and injection of dependency .